packages/contracts-bedrock/test/DeployImplementations.t.sol

```diff
@@ -245,6 +258,106 @@ contract DeployImplementationsOutput_Test is Test {
+            L1CrossDomainMessenger(expOutToml.readAddress(".dio.l1CrossDomainMessengerImpl"));
+        L1ERC721Bridge l1ERC721BridgeImpl = L1ERC721Bridge(expOutToml.readAddress(".dio.l1ERC721BridgeImpl"));
+        L1StandardBridge l1StandardBridgeImpl =
+            L1StandardBridge(payable(expOutToml.readAddress(".dio.l1StandardBridgeImpl")));
+        OptimismMintableERC20Factory optimismMintableERC20FactoryImpl =
+            OptimismMintableERC20Factory(expOutToml.readAddress(".dio.optimismMintableERC20FactoryImpl"));
+        DisputeGameFactory disputeGameFactoryImpl =
+            DisputeGameFactory(expOutToml.readAddress(".dio.disputeGameFactoryImpl"));
+
+        // Etch code at each address so the code checks pass when settings values.
+        vm.etch(address(opsmProxy), address(new Proxy(address(0))).code);
+        address opsmImpl = address(makeAddr("opsmImpl"));
+        vm.prank(address(0));
+        Proxy(payable(address(opsmProxy))).upgradeTo(opsmImpl);
+        vm.etch(address(opsmImpl), hex"01");
+        vm.etch(address(delayedWETHImpl), hex"01");
+        vm.etch(address(optimismPortalImpl), hex"01");
+        vm.etch(address(preimageOracleSingleton), hex"01");
+        vm.etch(address(mipsSingleton), hex"01");
+        vm.etch(address(systemConfigImpl), hex"01");
+        vm.etch(address(l1CrossDomainMessengerImpl), hex"01");
+        vm.etch(address(l1ERC721BridgeImpl), hex"01");
+        vm.etch(address(l1StandardBridgeImpl), hex"01");
+        vm.etch(address(optimismMintableERC20FactoryImpl), hex"01");
+        vm.etch(address(disputeGameFactoryImpl), hex"01");
+
+        // Manually set the values in the DeployImplementationsOutput contract.
+        dio.set(dio.opsmProxy.selector, address(opsmProxy));
+        dio.set(dio.delayedWETHImpl.selector, address(delayedWETHImpl));
+        dio.set(dio.optimismPortalImpl.selector, address(optimismPortalImpl));
+        dio.set(dio.preimageOracleSingleton.selector, address(preimageOracleSingleton));
+        dio.set(dio.mipsSingleton.selector, address(mipsSingleton));
+        dio.set(dio.systemConfigImpl.selector, address(systemConfigImpl));
+        dio.set(dio.l1CrossDomainMessengerImpl.selector, address(l1CrossDomainMessengerImpl));
+        dio.set(dio.l1ERC721BridgeImpl.selector, address(l1ERC721BridgeImpl));
+        dio.set(dio.l1StandardBridgeImpl.selector, address(l1StandardBridgeImpl));
+        dio.set(dio.optimismMintableERC20FactoryImpl.selector, address(optimismMintableERC20FactoryImpl));
+        dio.set(dio.disputeGameFactoryImpl.selector, address(disputeGameFactoryImpl));
+
+        // File must end with a new line because forge-std/StdToml adds a new line at the end of the file.
+        string memory actOutPath = string.concat(root, "/.testdata/test-deploy-implementations-output.toml");
+        // forge-std/StdToml serializes TOML key-value pairs in alphabetical order when writing to a TOML file.
```

Comment:
nit, technically it's foundry that does this, since foundry implements the cheat codes
```suggestion
        // File must end with a new line because Foundry adds a new line at the end of the file.
        string memory actOutPath = string.concat(root, "/.testdata/test-deploy-implementations-output.toml");
        // Foundry serializes TOML key-value pairs in alphabetical order when writing to a TOML file.
```